README.md

```diff
@@ -284,7 +284,7 @@ Now that we configured the Data Factory, we need to create the train pipeline th
       1. Select **New** 
       1. Configure the parameter
          1. Name: train_type
-         2. Value: adf_pipeline
+         2. Value: register_model
```

Comment:
In het train notebook is dit de waarde die 'train_type' moet hebben om een model te registreren. Dat willen we wel, dus moeten we hier niet dit hebben ipv 'adf_pipeline'?

Reply:
Ja, dit was hiervoor adf_pipeline. Dus volgens mij heb je het goed aangepast.